app/helpers/application_helper.rb

```diff
@@ -107,4 +107,28 @@ def link_to_article_search
       articles_path(mapped_params)
     )
   end
+
+  # TODO: add bento link
+  def link_to_bento_search
+    mapped_params = { q: params[:q] }
+    link_to(
+      t('searchworks.search_dropdown.bento.description_html'),
+      'https://library.stanford.edu/'
+    )
+  end
+
+  def link_to_library_website_search
+    link_to(
+      t('searchworks.search_dropdown.library_website.description_html'),
+      "https://library.stanford.edu/search/all?search=#{params[:q]}"
+    )
+  end
+
+  def search_type_name
+    if controller_name == 'articles'
```

Comment:
@jcoyne, I talked this one over with @jkeck and the end result looks like this: 
```
def search_type_name
  t("searchworks.search_dropdown.#{controller_name}.label")
end
``` 
The catalog / articles+ distinction is used in a couple different places throughout the app. Some of them are baked-in HTML (and it would probably be good to refactor at some point...)